### ISSUE TITLE
Settings, edit, and add new buttons are difficult to understand for screen reader users

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1580]
## Environment
NVDA 2017

## Problem
Screen reader users "can't make sense" of toolbar labeling.

![cant-understand-toolbar](https://user-images.githubusercontent.com/40970507/50322568-2c3abe00-049b-11e9-80fd-43a1f1beebed.gif)

**Comment by joshblack** [comment]
@dakahn is this just in the storybook? I think product authors would add appropriate labelling to indicate what they are for.

Is there something structural that we need to add for the toolbar in terms of markup or JS behavior?

**Comment by dakahn** [comment]
Example labeling could be improved. Since the tool bar comes before the content in the tab order and dom order (for screen readers) it's difficult to understand what each control does.

**Comment by dakahn** [comment]
we're working on improved accessibility spec and documentation. we can close this out
